src/server/middleware/api/github.js

```diff
@@ -0,0 +1,67 @@
+const { graphql } = require('@octokit/graphql')
+const NodeCache = require('node-cache')
+
+const { isProduction } = require('../../utils')
+
+const cache = new NodeCache({ stdTTL: 900 })
+
+async function getFreshGithubData() {
+  try {
+    const { repository } = await graphql(
+      `
+        {
+          repository(owner: "iterative", name: "cml") {
+            stargazers {
+              totalCount
+            }
+          }
+        }
+      `,
+      {
+        headers: {
+          authorization: `token ${process.env.GITHUB_TOKEN}`
```

Comment:
For the token, I created a personal access token from my account.

Reply:
could we take the one that we use in dvc.org deployment?

Reply:
Sure! Variable is updated!